src/pages/guides/using/creating_actions.md

```diff
@@ -186,9 +188,70 @@ function main(params) {
 
 exports.main = main;
 ```
-
 You can also set cookies or cache control headers, perform a HTTP redirect, and so forth.
 
+### Unsuccessful response 
+
+On failed web action invocations, the error code and message should be wrapped in an `error` object, as this would allow the system to interpret the response as an `applicationError`. 
+
+```javascript
+function main(params) {
+    try {
+        throw new Error("Boom!")
+    } catch (err) {
+        return {
+            error: {
+                statusCode: 500,
+                body: {
+                    payload: `Something went wrong: ${err}`
```

Comment:
I'd call it `message` instead of `payload` to be aligned with common REST practics. But I don't think it's that critical.

Reply:
Its updated.